zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -5240,6 +5240,9 @@ components:
         name:
           type: string
           description: The name of the mapping.
+        id:
```

```suggestion
        mappingId:
```

🔧 Please qualify IDs and keys for clarity.